src/components/Pool/LoadingGrid/LoadingGrid.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { useResponsiveItemsPerPage } from '../../../hooks/useResponsiveItemsPerPage'
+import { LoadingCard } from './LoadingCard'
+
+interface LoadingGridProps {
+  itemsAmount?: number
+}
+
+export function LoadingGrid({ itemsAmount }: LoadingGridProps) {
+  const responsiveItemsAmount = useResponsiveItemsPerPage()
+
+  return (
+    <GridLayout>
+      {new Array(itemsAmount && itemsAmount !== 0 ? itemsAmount : responsiveItemsAmount).fill(null).map((_, index) => (
```

Comment:
It needs a value otherwise the linter complains

<img width="696" alt="image" src="https://user-images.githubusercontent.com/5664434/175285940-abe65fc7-9625-481b-b655-ac02fc508ebc.png">

Reply:
a ye, my bad I checked that code with js not ts. Ignore it